packages/docs/src/pages/en/getting-started/contributing.md

```diff
@@ -54,120 +66,59 @@ yarn
-
-## Commit Guidelines with Commitizen
-
-Commitizen provides a fluid interface for handling semantic versioning. This provides a great boilerplate making it easier to write patch notes.
+### Choosing a base branch
 
-All commits should use [commitizen](https://github.com/commitizen/cz-cli) with the [conventional-changelog](https://github.com/conventional-changelog/conventional-changelog) as noted above. Commits should follow the format `<type>: <subject>` or `<type>(scope): <subject>`
+Before starting development you should know which branch to base your changes on. If in doubt use master as changes to master can usually be merged into a different branch without rebasing.
 
-### Commit types
-
-- **feat:** Commits that result in a new features or functionalities. Backwards compatible features will release with the next **MINOR** whereas breaking changes will be in the next **MAJOR**. The body of a commit with breaking changes must begin with `BREAKING CHANGE`, followed by a description of how the API has changed.
-- **fix:** Commits that provide fixes for bugs within vuetify's codebase.
-- **docs:** Commits that provide updates to the docs.
-- **style:** Commits that do not affect how the code runs, these are simply changes to formatting.
-- **refactor:** Commits that neither fixes a bug nor adds a feature.
-- **perf:** Commits that improve performance.
-- **test:** Commits that add missing or correct existing tests.
-- **chore:** Other commits that don't modify src or test files.
-- **revert:** Commits that revert previous commits.
-
-## Submitting Changes / Pull Requests
-
-Before doing any commits, you will want to pull down the latest and greatest from dev. From here, merge, and resolve any conflicts between your branch and dev. Its a good rule of thumb to pull frequently as development is constantly happening.
-
-In git, add all relevant files.
-
-Commit with commitizen using the command `git cz`. From here you will follow through a series of props. Make sure to select the appropriate type (see **Commit Guidelines w/Commitizen** above)
-
-Lastly, `git push` and open a pull request.
-
-### Pull Requests For Vuetify
-
-<alert type="info">
-
-  Pull requests related to Vuetify:
-
-- For **bug fixes** and **documentation updates** submit pull requests to `master`.
-- For **new features** and **enhancements** submit pull requests to `dev`
-- For **bugs** and **critical fixes** related to **v1.5/LTS** submit pull requests to `stable`
-- For any **features** that contain **breaking changes** submit pull requests to `next`
-
-</alert>
-
-### Pull Requests For Docs
-
-<alert type="info">
-
-  For any pull requests related to Vuetify docs, submit your pull request to the `master` branch.
-
-</alert>
+| Type of change | Branch |
+| --- | --- |
+| Documentation | `master` |
+| Bug fixes | `master` |
+| New features | `dev` |
+| Features with breaking changes | `next` |
+| Bugs and critical fixes for v1.5/LTS | `stable` |
```

Comment:
do we need to mention stable? it's not maintained

Reply:
Stable is a silly name anyway, I'll probably call it `2.4-LTS`